dev-util/clion/clion-2017.3.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```

Comment:
inherit desktop

eutils doesn't have what you need anymore.

Reply:
sure?
 
why does it work?

``` 
$ equery f clion
 * Searching for clion ...
 * Contents of dev-util/clion-2017.3.1:
...
/usr/bin/clion
/usr/share/applications/clion-clion.desktop
/usr/share/pixmaps/clion.svg
...
```

Reply:
eutils inherits desktop for existing EAPIs (0 - 6).

When this ebuild is moved to EAPI7, it'll stop working.

Reply:
alright, added `inherit desktop`.
`inherit eutils` is still required for `make_wrapper`